### PR TITLE
added a nexmo.js to the controllers folder

### PR DESCRIPTION
**Pull request by abdullateef97**
added a nexmo module to controller, it allows u send sms using the nexmo platform

### CODE REVIEW

**Review by mjrdnk — CHANGES_REQUESTED** [review]
Thank you very much for your contribution @abdullateef97 ! Possible that you create 'Examples' section in README (between Usage and Credits) and move existing example controller to 'Examples' section in README?

File: controllers/nexmo.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    this.apiSecret = apiSecret;
+    this.nex = new Nexmo({
+        apiKey : this.apiKey,
+        apiSecret : this.apiSecret
+    })
+}
+
+
+
+nexmo.prototype.sendSms = (from,toPath,message)=>{
```
Comment (mjrdnk) [inline controllers/nexmo.js]: Is it possible to rewrite these methods in the way to avoid .prototype? 
Thanks!

File: controllers/nexmo.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    
+    // toPath = file absolute path
+
+            this.nex.message.sendSms(from, to, message, (err,responseData)=>{
```
Comment (mjrdnk) [inline controllers/nexmo.js]: make sure you are using single spaces between 
```
)=>{
```
 
so it becomes 
```
) => {
```

Just for the sake of the same style all around the project.

**Review by mjrdnk — CHANGES_REQUESTED** [review]
Please, do the last adoptions:
1. remove code from controller (I would like to keep the files clean - examples only in README),
2. remove changes from package.json (same as above),
3. remove changes from 'usage' section (this section is for express-simple usage),
4. Make sure to keep the same white spaces stylings as in the rest of the project (check files how I used spaces between `() => {}` )

Thanks

**Comment by abdullateef97** [comment]
@mjrdnk where do i add the usage

File: controllers/nexmo.js (hunk 1 above)
Comment (abdullateef97) [reply controllers/nexmo.js]: i already did that

Comment (abdullateef97) [reply controllers/nexmo.js]: where do i add the usage

**Comment by mjrdnk** [comment]
For usage write a snippet for server.js and route for this controller.
So **only** in README file we need something like this:

**Examples:**

Example: nexmo - sms service 
To write your custom controllers write your functions in the file and export the module:

```
here goes controller for nexmo
```

so the controller could be reused in router file nexmo.route.js, like this (please check other helloWorld route for implementation): 

```
here goes route for nexmo
```

In order to use it in your app you have to declare it in the server.js, like this:
```
...
let API_BASE = config.API_BASE;
let helloWorld = require('./routes/helloWorld.js');
let nexmo = require('./routes/nexmo.route.js');

app.use(function(req, res, next) {
...

...

}));

app.use(API_BASE, helloWorld);
app.use(API_BASE, nexmo);

app.listen(port, function() {
...
```
After that, your new nexmo route will be available in the REST call.

If you have further questions, please ask.

**Comment by abdullateef97** [comment]
everything shd be in the readme file

**Comment by mjrdnk** [comment]
Yes, this use case (nexmo) is so specific that it's place should be in the README examples section. However it would be the best to have some more generic example, but I will accept it anyway.

**Comment by abdullateef97** [comment]
@mjrdnk i believe my new commit should meet your specs

**Review by mjrdnk — APPROVED** [review]
Good Job, thanks for your awesome contribution! How would you use this endpoint /sms? Send sms as the body?

**Comment by mjrdnk** [comment]
Just saw that markup is broken after merging. Please fix.

**Comment by abdullateef97** [comment]
can u specify what part

**Comment by mjrdnk** [comment]
I fixed it already. Thanks for your contribution 😊

**Pull request merged**